packages/vuetify/src/directives/colorable/__tests__/colorable.spec.ts

```diff
@@ -0,0 +1,241 @@
+// Libraries
+import Vue from 'vue'
+
+// Directives
+import Colorable from '../'
+
+// Utilities
+import {
+  mount,
+  Wrapper,
+} from '@vue/test-utils'
+
+interface BorderModifiers {
```

Comment:
@vuetifyjs/core-team @vuetifyjs/contributors thoughts on process for v3? Should interfaces have a more defined place and if so, where?

Reply:
They are already defined here https://github.com/vuetifyjs/vuetify/pull/10866/files/f00d2896fa6a2f2da57817b5f005ff6e6c5c66da#diff-ec2fd9fa5f8e2ca617cce868d0686c54R8 (and in this particular case it's a good place)

Reply:
That's the exact line this conversation is already referencing. Why in file? That seems like a bad pattern for more than this use-case.

Reply:
I'm going to look at using api-extractor like vue does.